aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -233,7 +246,7 @@ module aptos_framework::account {
                 key_rotation_events,
                 rotation_capability_offer: CapabilityOffer { for: option::none() },
                 signer_capability_offer: CapabilityOffer { for: option::none() },
-            }
+            },
```

Comment:
@robinlzw, do you remember there are comments about this added comma?

Reply:
Of course, I'll fix this today first, it's a simple issue.